website/docs/r/org_oidc.html.markdown

```diff
@@ -0,0 +1,214 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_org_oidc"
+sidebar_current: "docs-vcd-resource-org-oidc"
+description: |-
+  Provides a resource to configure OpenID Connect (OIDC) for an Organization in VMware Cloud Director.
+---
+
+# vcd\_org\_oidc
+
+Provides a resource to configure OpenID Connect (OIDC) for an Organization in VMware Cloud Director.
+
+Supported in provider *v3.13+*
+
+## Example Usage with Well-known Configuration Endpoint
+
+The well-known configuration endpoint retrieves all the OpenID Connect settings values:
+
+```hcl
+data "vcd_org" "my_org" {
+  name = "my-org"
+}
+
+resource "vcd_org_oidc" "oidc" {
+  org_id                 = data.vcd_org.my_org.id
+  enabled                = true
+  prefer_id_token        = false
+  client_id              = "clientId"
+  client_secret          = "clientSecret"
+  max_clock_skew_seconds = 60
+  wellknown_endpoint     = "https://my-idp.company.com/oidc/.well-known/openid-configuration"
+}
+```
+
+Users can override any value retrieved by the well-known configuration:
+
+```hcl
+data "vcd_org" "my_org" {
+  name = "my-org"
+}
+
+resource "vcd_org_oidc" "oidc" {
+  org_id                 = data.vcd_org.my_org.id
+  enabled                = true
+  prefer_id_token        = false
+  client_id              = "clientId"
+  client_secret          = "clientSecret"
+  max_clock_skew_seconds = 60
+  wellknown_endpoint     = "https://my-idp.company.com/oidc/.well-known/openid-configuration"
+
+  # Overrides:
+  access_token_endpoint = "https://my-other-idp.company.com/oidc/token"
+  userinfo_endpoint     = "https://my-other-idp.company.com/oidc/userinfo"
+}
+```
+
+Once the OIDC settings are created, if users want to restore an overridden value to the original one given by the
+well-known configuration endpoint, they must perform an update in code to set the previous value explicitly:
+
+```hcl
+data "vcd_org" "my_org" {
+  name = "my-org"
+}
+
+resource "vcd_org_oidc" "oidc" {
+  org_id                 = data.vcd_org.my_org.id
+  enabled                = true
+  prefer_id_token        = false
+  client_id              = "clientId"
+  client_secret          = "clientSecret"
+  max_clock_skew_seconds = 60
+  wellknown_endpoint     = "https://my-idp.company.com/oidc/.well-known/openid-configuration"
+
+  access_token_endpoint = "https://my-idp.company.com/oidc/token"          # Restores the previous value
+  userinfo_endpoint     = "https://my-other-idp.company.com/oidc/userinfo" # Still overridden
+}
+```
+
+In other words, removing the argument or setting it to `""` won't make the original value from the well-known configuration endpoint
```

Comment:
I would highlight "won't" as it's easy to miss while quickly reading.

Reply:
Changed, thanks!